src/components/page-environment/environment-overview.js

```diff
@@ -130,55 +124,64 @@ export class EnvironmentOverview extends React.Component {
               <React.Fragment>
                 <div className="env__content">
                   <section className="grid">
```

Comment:
add class `grid--gap-medium` to provide spacing
```suggestion
                  <section className="grid grid--gap-medium">
```